x/ibc/light-clients/solomachine/types/consensus_state.go

```diff
@@ -51,6 +53,9 @@ func (cs ConsensusState) ValidateBasic() error {
 	if cs.Timestamp == 0 {
 		return sdkerrors.Wrap(clienttypes.ErrInvalidConsensus, "timestamp cannot be 0")
 	}
+	if cs.Diversifier != "" && strings.TrimSpace(cs.Diversifier) {
```

Comment:
@cwgoes not sure if we need to do another validation for the diversifier

Reply:
no, I think this is fine